<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Myplayer</class>
 <widget class="QWidget" name="Myplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Myplayer-这风吹得我真冷</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="pre_bt">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>上一首</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Myplayer/pre.jpg</normaloff>../Myplayer/pre.jpg</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="play_bt">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>   播放</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Myplayer/2.jpg</normaloff>../Myplayer/2.jpg</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="next_bt">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>下一首</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Myplayer/next.jpg</normaloff>../Myplayer/next.jpg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="username">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>用户名：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="username_text">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>NoneUser</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logout_bt">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>登出</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Myplayer/logout.jpg</normaloff>../Myplayer/logout.jpg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="nowplay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>正在播放：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nowplay_info">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>20</pointsize>
              </font>
             </property>
             <property name="text">
              <string>没在播放呢</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>进度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="nowplay_posi">
             <property name="singleStep">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="vulume_2">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="text">
              <string>音量</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="vulumesilider">
             <property name="cursor">
              <cursorShape>OpenHandCursor</cursorShape>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
             <property name="sliderPosition">
              <number>20</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>选择歌单：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="songlist_choice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>5</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mode_choice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>顺序播放</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>循环播放</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>随机播放</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>下载进度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="download_bar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="serach_input"/>
       </item>
       <item>
        <widget class="QPushButton" name="serach_bt">
         <property name="font">
          <font>
           <family>AcadEref</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>搜索</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../Myplayer/serach.jpg</normaloff>../Myplayer/serach.jpg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="songlist"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CopyRight  Reserved   by 这风吹得我真冷</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>赏口饭（donate）</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../Myplayer/donate.jpg</normaloff>../Myplayer/donate.jpg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
